platforms/ios/build_framework.py

```diff
@@ -130,7 +130,12 @@ def _build(self, outdir):
                 cmake_flags.append("-DCMAKE_EXE_LINKER_FLAGS=" + " ".join(c_flags))
 
                 # CMake cannot compile Swift for Catalyst https://gitlab.kitware.com/cmake/cmake/-/issues/21436
-                # cmake_flags.append("-DCMAKE_Swift_FLAGS=" + " " + target_flag)
+                import shutil
+                swift_compiler = shutil.which("swift")
+                if swift_compiler:
+                    cmake_flags.append(f"-DCMAKE_Swift_COMPILER={swift_compiler}")
+                else:
+                    print("Error: Swift compiler not found!")
```

Comment:
The default swift compiler in PATH is found by CMake automatically the patch does not make sense.